build/Dockerfile

```diff
@@ -141,11 +141,13 @@ RUN --mount=type=cache,target=/home/${USER}/.cache/pip,uid=${USER_UID} \
 # fms_acceleration_peft = PEFT-training, e.g., 4bit QLoRA
 # fms_acceleration_foak = Fused LoRA and triton kernels
 # fms_acceleration_aadp = Padding-Free Flash Attention Computation
+# fms_acceleration_moe = Scatter Mixture of Experts models
 RUN if [[ "${ENABLE_FMS_ACCELERATION}" == "true" ]]; then \
         python -m pip install --user "$(head bdist_name)[fms-accel]"; \
         python -m fms_acceleration.cli install fms_acceleration_peft; \
         python -m fms_acceleration.cli install fms_acceleration_foak; \
         python -m fms_acceleration.cli install fms_acceleration_aadp; \
+        python -m fms_acceleration.cli install fms_acceleration_moe; \
```

Comment:
Are we enabling fms_acceleration_moe to the image released even though it is experimental and requires post-processing for inference to work? This seems like this feature is ready to be available for people to use/experiment with, but not ready for full production release